### ISSUE TITLE
Add application logs section when using tokio-console

### ISSUE THREAD

**Issue by Milo123459** [tokio-rs/console#297]
Something to see & filter the logs from your application.

**Comment by hawkw** [comment]
Yeah, we'd definitely like to eventually have a way to stream a log of all `tracing` events from the program. One thing we've also considered is adding a way to filter event logs by task, and display a log of all the events that occurred within a particular task in the Task Details view. Of course, we could also show event logs for all tasks,

Some design considerations:
- This should probably be a separate RPC than the main [`Instrument::watch_updates`](https://docs.rs/console-api/latest/console_api/instrument/instrument_server/trait.Instrument.html#tymethod.watch_updates) RPC. We're already streaming a lot of data from the instrumented application, and most of the views the console currently provides probably won't display logs by default. Therefore, we'll probably want to have a way to toggle on and off log streams separately from other data streams, so that we're only streaming them as needed.
- How would we want to filter what logs are displayed?
- Since we can stream logs as structured data rather than as strings, we can probably display them in some much more interesting ways than just a bunch of lines of text --- e.g. some data can be conditionally hidden or shown. However, we probably want to start with something simple and then iterate on it.

**Comment by Milo123459** [comment]
Filtering should just be pressing ctrl+f and showing a text box
